### PR TITLE
Get data for the managing body endpoint.

### PR DESCRIPTION
**Pull request by Lemore**
#81

### CODE REVIEW

**Comment by nirgn975** [comment]
@Lemore Are you finish this PR?

**Comment by Lemore** [comment]
I think so

**Comment by nirgn975** [comment]
@Lemore Please re-roll from master, you have a lot of files that for somehow looks like you add them (buy they already at master, from previous PR).

File: server/pension/urls.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from django.conf.urls import include, url
```
Comment (nirgn975) [inline server/pension/urls.py]: @Lemore This file is already exist in `master`, pls re-roll.

File: server/dummy_data.csv [hunk 2]
```diff
@@ -0,0 +1,101 @@
+﻿managing_body,fund,fund_name,report_year,report_qurater,instrument_type,instrument_sub_type,instrument_id,orig_instrument_id,instrument_name,issuer,activity_industry,currency,fair_value,market_cap,rate_of_fund,rating_agency,reference_index,intrest_rate,date_of_purchase,average_of_duration,date_of_revaluation,rate,yield,rating,par_value,tmp_name,underlying_asset,type_of_asset,rate_of_ipo,liquidity,asset_type
+clal,99078,"כלל-אג""ח ופקדונות***",2015,2,ניירות ערך סחירים,"אג""ח קונצרני",7480072,IL0074800728,דיסקונט מנ הת ח,דיסקונט,בנקים,NIS,5244248.17,0,0.292277408,מעלות,null,4.29,null,1.18,null,123.5,0.17,AA-,4246354.79,null,null,null,0.747930007,סחיר,"אג""ח קונצרני"
```
Comment (nirgn975) [inline server/dummy_data.csv]: This file is already exist in `master`, pls re-roll.

File: __init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (nirgn975) [inline __init__.py]: @Lemore I'm not sure way you commit this file, there is no need for one.

File: server/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (nirgn975) [inline server/__init__.py]: @Lemore I'm not sure way you commit this file, there is no need for one. Pls remove it.

**Comment by Lemore** [comment]
@nirgn975 Ready for pull

**Pull request closed without merging**